src/blog/lagom-1-5-0-M1.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Lagom 1.5.0-M1 released!
+date: 2018-07-25
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.5.0-M1
+---
+
+We are happy to announce the availability of Lagom 1.5.0-M1. This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and starts moving towards supporting scala 2.13.
+
+The motivation behind moving Lagom closer to Play and Akka is twofold: Play and Akka users will be able to adopt Lagom more easily by reusing existing code or APIs they are already familiar with, and framework maintainers will reduce the amount of code to maintain and converge in conventions speeding up the development process.
+
+## Highlights
```

Comment:
Let's use sub-headings under this section, rather than bullet points. Each one is a bit long for a bullet list.